src/types/interfaces.ts

```diff
@@ -598,6 +600,7 @@ export interface IAcceptSessionRequest extends ISdkMediaDeviceIds {
 export interface IEndSessionRequest {
   conversationId: string;
   reason?: Constants.JingleReasonCondition;
+  fromHeadset?: boolean;
```

Comment:
I know this wasn't part of your PR, but could you add some type docs to all these props in this interface? They are pretty self explanatory, but it is still nice to have the type docs for text editors.

Reply:
Sure no problem! There were a few other interfaces that used similar props, should I just base it off those for this one?

Reply:
Yeah, you can borrow from those.